app-modules/interaction/graphql/interaction.graphql

```diff
@@ -145,11 +160,10 @@ input UpdateInteractionInput {
     division_id: UUID @rules(apply: ["exists:divisions,id"])
 
     "The start datetime of the interaction."
-    start_datetime: DateTime @rules(apply: ["date_format:Y-m-d H:i:s"])
+    start_datetime: DateTime @rules(apply: ["date:Y-m-d H:i:s"])
```

Comment:
Was this still giving you problems? I don't think that `date:Y-m-d H:i:s` is a valid rule in Laravel.

Reply:
I put it back and It appears to be working now. It might have been a bug with the autosubscriptions causing it to fail.